doc/setup.md

```diff
@@ -0,0 +1,130 @@
+* scikit-image >= 0.14.0
+* CGAL >= 4.13
+* numpy
+ 
+##  Setup with linux 
+
+### Geocruncher
+
+1. run the following key to clone the geocruncher repository:
+
+        git clone https://github.com/ISSKA/geocruncher.git
+
+2. To be able to run the python script you need to manually change some files within geoCruncher. 
+
+	Within \_\_Main\_\_.py change:
+
+        from . import main
+
+	to:
+
+        import main
+
+	and within Main.py change:
+
+        from topography_reader import txt_extract
+        from ComputeIntersections import CrossSectionIntersections, MapIntersections, GeocruncherJsonEncoder, Slice
+        from MeshGeneration import generate_volumes
+
+	to:
+
+        from .topography_reader import txt_extract
+        from .ComputeIntersections import CrossSectionIntersections, MapIntersections, GeocruncherJsonEncoder, Slice
+        from .MeshGeneration import generate_volumes
+
+	To run locally the . before imports needs to be deleted.
+
+### numpy
+
+1. run the following key:
+	
+        pip install numpy
+
+2. Or for python3
+	
+        pip3 install numpy
+
+### Eigen
+
+1. Download the eigen3 library in tar.gz format at their website (http://eigen.tuxfamily.org/index.php?title=Main_Page) or using this link directely:
+	
+        http://bitbucket.org/eigen/eigen/get/3.3.7.tar.gz
+
+2. Create a folder named eigen-build.
+
+3. Run cmake-gui, select the original eigen folder for the source code and eigen-build to build binaries. Then configure and generate.
+
+4. If the generation have been succesful, go to the eigen-build folder and run:
+	
+        make install .
+
+### GmLib
+
+1. run the following key to clone the gmlib repository:
+
+        git clone https://gitlab.inria.fr/gmlib/gmlib.git
+
+2. You will need to enter your Gitlab INRIA username and password.
+
+3. Within your git repository run the following key:
+	
+        pip install -e .
+
+Or for python3
+	
+        pip3 install -e .
+
+From now on you can compute intersections, however to compute meshes you still need two package CGAL and Meshtools
+
+### CGAL
+
+1. run the following key to clone the cgal repository:
+	
+        git clone https://github.com/CGAL/cgal.git
+
+2. Create a folder named cgal-build.
+
+3. Run cmake-gui, select the original cgal folder for the source code and cgal-build to build binaries. Then configure and generate.
+
+4. Tick yes for CGAL_HEADER_ONLY and CGAL_DEV_MODE options.
+
+5. If the generation have been succesful, go to the cgal-build folder and run:
+
+        make install .
+
+### Meshtools
+
+1. run the following key to clone the gmlib repository:
+	
+	git clone https://gitlab.inria.fr/charms/MeshTools.git
+
+2. You will need to enter your Gitlab INRIA username and password.
+
+3. Setup the path to your cgal location usually by runnning:
+	
+	export CGAL_DIR=/usr/local/lib/cmake/CGAL
+
+4. Within your git repository run the following key:
+	
+        pip install .
+
+Or for python3
+	
+        pip3 install .
```

Comment:
Also here, ,the installation is described on gitlab. And at least on Windows I didn't know of pip3, I just use pip

Reply:
pip3 is for python 3 simply ^^

Reply:
That's clear. I guess only relevant if you have both python 2 & 3 installed